source/_patterns/02-organisms/components/highlight~article-long-authors.json

```diff
@@ -0,0 +1,23 @@
+{
+  "heading": {
+    "heading": "Highlights"
+  },
+  "items": [
+    {
+      "title": "[Do not use this base pattern directly: see the following variants of this pattern for usage]",
+      "url": "#link-to-article",
+      "authors": "Yongjian Huang, Shashank Bharill, John Kuriyan, Lee R Berger, John Hawks, Bernhard Zipfel, Ming Chao",
+      "summary": "Summary",
+      "meta": {
+        "text": "#"
+      },
+      "image": {
+        "fallback": {
+          "defaultPath": "https://unsplash.it/250/140/",
+          "altText": "Meaningful alt text please."
+        }
+      }
+    }
+  ],
```

Comment:
The variants for highlight should be against highlight-item.

Reply:
If these variants are against highlight-item, I can add only one item for each type. Is that fine?

Reply:
Done, all examples are created as a highlight-item